File: Latihan B/java/Disk.java
// deklarasi kelas disk
public class Disk{
    // variabel privat
    private String type;
    private String capacity;
    private int price;
    // method public dan konstruktor
    public Disk(){

    }
    public Disk(String type, String capacity, int price){
        this.type=type;
        this.capacity=capacity;
        this.price=price;
    }
    // get dans set semua variabel
    public String gettype(){
        return type;
    }
    public void settype(String type){
        this.type = type;
    }
    public String getcapacity(){
        return capacity;
    }
    public void setcapacity(String capacity){
        this.capacity = capacity;
    }
    public int getprice(){
        return price;
    }
    public void setprice(int price){
        this.price = price;
    }
}

File: Latihan B/python/Disk.py
# deklarasi kelas
class Disk:
    # deklarasi variabel
    __type = ""
    __capacity = ""
    __price = 0
    # method dan konstruktor
    def __init__(self, type ="", capacity= "", price = 0):
        self.__type = type
        self.__capacity = capacity
        self.__price = price
    #  get dans set semua variabel
    def setprice(self, price):
        self.__price = price
    def getprice(self):
        return self.__price
    def settype(self, type):
        self.__type = type
    def gettype(self):
        return self.__type
    def setcapacity(self, capacity):
        self.__capacity = capacity
    def getcapacity(self):
        return self.__capacity